pom.xml

```diff
@@ -401,5 +427,50 @@
                 <module>jersey-tck</module>
             </modules>
         </profile>
+
+        <!-- This profile is activated when Java 21 or later is used to build and the project has a src/main/java21
+             directory. -->
+        <profile>
```

Comment:
I am -1 for making this dependend of the used JDK. I agreed solely to *adding* 17 into the MR, not to making 21 *optional*-

Reply:
No worries, I can remove that. This was just setup in case there was any need for an MR JAR. It's easily removed and serves no purpose yet anyway.